pkg/apis/flowcontrol/install/install.go

```diff
@@ -27,7 +27,12 @@ import (
 )
 
 func init() {
-	Install(legacyscheme.Scheme)
+	Install(GetTheScheme())
+}
+
+// GetTheScheme returns the `*Scheme` with which this package registers stuff
+func GetTheScheme() *runtime.Scheme {
```

Comment:
I don't really understand how this helps things? Do we do this other places?

Reply:
This is in response to a concern that @yue9944882 raised about the conversion code in `validation.go`.  He said we aim to remove legacyscheme, which was directly referenced from `validation.go`.  By accessing the proper Scheme through this procedural abstraction, this makes it easy to update both references at once when that day comes.

Reply:
we are avoiding the references to legacyscheme in the codebase. i cant find the past conversation but i remember there used to be one giant `Scheme` in the past and somewhere it was saying that we're splitting the legacyscheme into multiple dedicated/private `Scheme` and removing the reference for legacyscheme to either move some projects out-of-tree or just sanitizing the codebase. at least for now, the legacy should be working only for round-tripping in the storage layer.

Reply:
I don't really think this helps much with that goal. But I am not asking for a change because this isn't very important compared to the other changes I've asked for.